.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         node-version: [14.x, 16.x, 18.x, 20.x]
     steps:
       - name: Checkout
-        uses: actions/checkout@v3.3.0
+        uses: actions/checkout@v4.1.1
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v3.6.0
```

Comment:
Did you consider upgrading `actions/setup-node`?

Reply:
Oh, I must have skipped that by accident, thanks for pointing it out!